task.go

```diff
@@ -0,0 +1,32 @@
+package mesg
+
+// Taskable represents implementation of task executor.
+type Taskable interface {
+	// Name returns the task key.
+	Name() (key string)
+
+	// Execute called when a task execution request arrived with the same key
+	// returned by Name().
+	// key is the output key and data is the output data of task.
+	Execute(execution *Execution) (key string, data Data)
```

Comment:
Can you add more context to the returns: eg: OutputKey, OutputData.

Reply:
done